Allow templating (#409)
semantic-release/gitlab: README.md

| Property | Description                                                                                                 | Default                              |
| -------- | ----------------------------------------------------------------------------------------------------------- | ------------------------------------ |
| `path`   | **Required**, unless `url` is set. A [glob](https://github.com/isaacs/node-glob#glob-primer) to identify the files to upload.    | -                                    |

> | `url`    | Alternative to setting `path` this provides the ability to add links to releases, e.g. URLs to container images. Can be dynamically adjusted with [Lodash template](https://lodash.com/docs#template). Allows same variables as [`successComment`](#successComment) + all given environment variables.                           | -                                    |

Reviewer: I don't like that we duplicate this text everywhere. What about adding a new section `Templating` and just have something like this here:

```suggestion
| `url`    | Alternative to setting `path` this provides the ability to add links to releases, e.g. URLs to container images. Supports [templating](#templating). | -                                    |
```

Author: @fgreinacher I added a link to the lodash docs (template section) (:
